config.go

```diff
@@ -49,7 +49,7 @@ const (
 	defaultRPCHost             = "localhost"
 	defaultMaxPendingChannels  = 1
 	defaultNoSeedBackup        = false
-	defaultTrickleDelay        = 30 * 1000
+	defaultTrickleDelay        = 90 * 1000
```

Comment:
`90 * time.Second`?

Reply:
that would break old configs I guess, but I agree it would be nice to make all flags use `time.Duration`.

Reply:
hmm, yeah that is unfortunate that it doesn't. we could migrate to a `trickleinterval` that uses `time.Duration` and deprecate `trickledelay`, though that can be done later